### DISCUSSION TITLE
Not able to add new meetings to Tennessee AA / Map Issue

### DISCUSSION THREAD

**Discussion by lionssharedigital** [code4recovery/12-step-meeting-list#1023] [Q&A] [answered]
We are having a problem with the maps again.  

When we are trying to put in a new meeting, once we put in an address, the does not maps to pull up and I am unable to publish a new meeting.  The only thing in the console is "Uncaught (in promise) Error: A listener indicated an asynchronous response by returning true, but the message channel closed before a response was received"

Also, I am no longer able to add a new region....says "Sorry, you are not allowed to edit terms in this taxonomy."

**Comment by joshreisner** [comment, marked as answer]
hi jesse, re the regions, we just fixed that in the latest update out last night, sorry about that.

regarding the other error, there are two ways to fix it:
1. keep using google maps but go into the google API console and enable the Geocoding API, or 
2. switch over to mapbox

see this topic for more info on that: https://github.com/code4recovery/12-step-meeting-list/discussions/1014#discussioncomment-4496125

**Reply by joshreisner** [reply]
glad to hear it's working now. if you had wanted to stay with google maps the next thing i'd look at is why you were seeing those `4XX` errors - possibly there were API restrictions? anyway mapbox is the best long-term solution anyway

**Comment by lionssharedigital** [comment]
Got it. I'll try Mapbox. 
I do have Geocoding enabled. It looks like all the requests are generating errors. 
https://tinyurl.com/2l7jnpl2
![download (1)](https://user-images.githubusercontent.com/10728016/210013472-dd700019-b141-49b4-bbc1-f935d78d4b4c.png)

**Comment by lionssharedigital** [comment]
Switching to Mapbox appears to work.
